apps/changelog/classes/release_version.py

```diff
@@ -0,0 +1,39 @@
+# Python Standard Library Imports
+import datetime
+from collections import namedtuple
+
+
+class ReleaseVersion(
+    namedtuple(
+        'ReleaseVersion',
+        'origin_url,ref,date,sha,branch',
+    )
+):
```

Comment:
# Reviewer Note
Python 2 compatible code to achieve `@dataclass` decorator in Python 3